### PR TITLE
Translations for es-fr-ja-pt

### PR DESCRIPTION
**Pull request by arnaud4d**
We would like to use these translations

### CODE REVIEW

File: codeTranslations/ja.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "cmfcmf/d-s-l.searchBar.placeholderWithoutVersion": "検索...",
+  "cmfcmf/d-s-l.searchBar.placeholderWithVersion": "検索... [{version}]",
+  "cmfcmf/d-s-l.searchBar.noResults": "該当する結果はありません。",
+  "cmfcmf/d-s-l.searchBar.clearButtonTitle": "Clear",
+  "cmfcmf/d-s-l.searchBar.detachedCancelButtonText": "Cancel",
+  "cmfcmf/d-s-l.searchBar.submitButtonTitle": "Submit"
```
Comment (cmfcmf) [inline codeTranslations/ja.json:7]: It looks like three translations are missing here.

**Review by cmfcmf — COMMENTED** [review]
Thank you!
Could you also add the newly supported languages to the Internationalization section of the README (https://github.com/cmfcmf/docusaurus-search-local#internationalization)?

File: codeTranslations/ja.json (hunk 1 above)
Comment (arnaud4d) [reply codeTranslations/ja.json:7]: our japanese translator was unsure if there would be enough room and proposed to keep EN words that are well understood

Comment (cmfcmf) [reply codeTranslations/ja.json:7]: I see, good to know 👍

**Pull request merged**